### DISCUSSION TITLE
Could I Build a starter on Linux using Intel Compiler?

### DISCUSSION THREAD

**Discussion by kjrstory** [OpenRadioss/OpenRadioss#2385] [Q&A] [answered]
Even though there are files under the starter/CMake_Compilers folder, such as cmake_linux64_intel.txt and cmake_linux64_intel_ifx.txt, there is no Intel Compiler in platforms.txt. Is it still possible to build an Intel compiler on Linux?

**Comment by OlivierWienholtz** [comment, marked as answer]
Hello, 

platforms.txt is only read in build_script.bash when invoking the help.
It prints examples for most commonly compiling uses.

Indeed : 
You can use 
./build_script.bash -arch=linux64_intel 
to build with Intel compiler legacy. 
For now we recommend this arch.

Hope it helps,
Olivier
